bbs/conf.cpp

```diff
@@ -460,7 +460,7 @@ char* GetGenderAllowed(int nGender, char* pszGenderAllowed) {
 /*
  * Function for editing the data for one conference.
  */
-int modify_conf(ConferenceType conftype, int which) {
+void modify_conf(ConferenceType conftype, int which) {
```

Comment:
and down here since it's not used outside the function, put "static" in front of void